src/main/java/io/quarkus/search/app/dto/SearchHit.java

```diff
@@ -1,9 +1,22 @@
 package io.quarkus.search.app.dto;
 
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.FieldProjection;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.HighlightProjection;
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IdProjection;
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.ProjectionConstructor;
 
-@ProjectionConstructor
-public record SearchHit(@IdProjection String id, @FieldProjection String title) {
+public record SearchHit(String path, String type, String title, String summary, Set<String> content) {
```

Comment:
Considering this is now all very specific to Guides... maybe let's rename this to `GuideSearchHit`? And move the REST API endpoint to `/guides/search`?

Reply:
makes sense 😃